packages/api/tests/lambdas/test-bulk-operation.js

```diff
@@ -157,6 +207,7 @@ test('bulk operation lambda throws error for unknown event type', async (t) => {
   }));
 });
 
+// This test must run for the following tests to pass
```

Comment:
Adding this because I was confused why `only` was failing. We need this test to set ENVs.

Reply:
Yeah, agreed. Leaving this comment open. I'll circle back and address after the feedback that's relevant to my changes.

Reply:
Here's a quick update that should fix it. The problematic test here is already deleting ENVs before trying to re-add them so I believe we can just add them normally in the `before` and expect the test to remove and add them.

https://github.com/nasa/cumulus/pull/2108/commits/01e8c77c48c0d0b824440494c0c879e1ba0468f9

Reply:
👍🏻  that works for now.